vhdbuilder/packer/components.json

```diff
@@ -293,7 +299,8 @@
       ],
       "multiArchVersions": [
         "v1.17.0",
-        "v1.18.0"
+        "v1.18.0",
```

Comment:
why keep 2 version.

Reply:
I don't think auto patch will remove a minor version, shall we delete it manually?

Reply:
I think it should be owner's call to remove another minor version from VHD, when they did a bump manually